client/my-sites/controller.js

```diff
@@ -373,14 +374,20 @@ export function siteSelection( context, next ) {
 		const state = getState();
 		const isAtomicSite = isSiteAutomatedTransfer( state, siteId );
 		const userCanManagePlugins = canCurrentUser( state, siteId, 'manage_options' );
-		const calypsoify = isAtomicSite && config.isEnabled( 'calypsoify/plugins' );
+		const calypsoifyPlugins = isAtomicSite && config.isEnabled( 'calypsoify/plugins' );
+		// TODO: add actual opt in setting
+		const calypsoifyGutenberg =
+			config.isEnabled( 'gutenberg' ) &&
```

Comment:
```suggestion
```

When we release this to production we'll be in a state where `gutenberg/opt-in` and `calypsoify/gutenberg` feature flags are set to true, but the `gutenberg` one should still remain falls - it is controlling the state of Gutenberg integration in Calypso which we don't want to enable yet.